Detectors/MUON/MCH/PreClustering/src/PreClusterFinder.cxx

```diff
@@ -88,27 +88,27 @@ void PreClusterFinder::reset()
 }
 
 //_________________________________________________________________________________________________
-void PreClusterFinder::loadDigits(const DigitStruct* digits, uint32_t nDigits)
+void PreClusterFinder::loadDigits(const Digit* digits, int nDigits)
```

Comment:
Hum. A (pointer, size) pair ... Looks a lot like like a `span<const Digit>` to me ... ;-)

Reply:
Agreed. In fact this is already done in the third commit that I just prepared. ;-)